extensions/amp-story/amp-story.md

```diff
@@ -345,7 +347,7 @@ In most implementations for live blogs, content is either pushed by the server t
 
 This means that publishers of stories do not need to set up a JSON endpoint or push mechanism for this functionality to work.
 
-Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](#data-poll-interval) attribute.
+Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](<#live-story-disabled-(optional)>) attribute.
```

Comment:
This is incorrect, the previous value was right?

Reply:
ah! great catch
the existing value was broken, but I duplicated the value being used elsewhere. pushed a fix